docs/source/concepts/deploy.rst

```diff
@@ -6,15 +6,73 @@ Deploying Bento
 Deployment Overview
 -------------------
 
-The three most common deployment options with BentoML are:
-
-- 🐳 Generate container images from Bento for custom docker deployment
-- `🦄️ Yatai <https://github.com/bentoml/Yatai>`_: Model Deployment at scale on Kubernetes
-- `🚀 bentoctl <https://github.com/bentoml/bentoctl>`_: Fast model deployment on any cloud platform
-
-
-Containerize Bentos
--------------------
+Here are the four most common deployment options with BentoML:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Deploy Method 
+     - Pros ❤️ 
+     - Cons 💔 
+   * - `🍱 BentoCloud <https://www.bentoml.com/>`_
+     - Serverless cloud for AI, fully managed, made for BentoML
+     - Limited availability, `sign up here <https://www.bentoml.com/bento-cloud/>`_ today for early access
+   * - Docker Containers 
+     - Flexible for deploying almost anywhere, easy to get started 
+     - Requires DevOps work for production deployment, lacks advanced scaling capabilities
```

Comment:
Rigid deployment as a single container. Low hardware utilization and inflexible scaling.

Reply:
how's "Rigid deployment as a single container. Inflexible for scaling and maximizing hardware utilization"?